E0611: No name '_ElementStringResult' in module 'lxml.etree'

In lxml 5.1.1 the private name `_ElementStringResult` in module `lxml.etree` does no longer exists.

This name is imported and used by `searx.utils.extract_text(..)`:

https://github.com/searxng/searxng/blob/f5bb64cca8a63a10c470b4f9bff647702b6de74c/searx/utils.py#L220-L221

- https://github.com/searxng/searxng/pull/3366 reports issue:

```
TEST      [pylint] ./searx ./searxng_extra ./tests
************* Module searx.utils
searx/utils.py:21:0: E0611: No name '_ElementStringResult' in module 'lxml.etree' (no-name-in-module)
```

--- a/searx/utils.py
+++ b/searx/utils.py
@@ -18,7 +18,7 @@ from urllib.parse import urljoin, urlparse
 from markdown_it import MarkdownIt
 
 from lxml import html
-from lxml.etree import ElementBase, XPath, XPathError, XPathSyntaxError, _ElementStringResult, _ElementUnicodeResult
+from lxml.etree import ElementBase, XPath, XPathError, XPathSyntaxError
 
 from searx import settings
 from searx.data import USER_AGENTS, data_dir
@@ -217,7 +217,7 @@ def extract_text(xpath_results, allow_none: bool = False) -> Optional[str]:
         text: str = html.tostring(xpath_results, encoding='unicode', method='text', with_tail=False)
         text = text.strip().replace('\n', ' ')
         return ' '.join(text.split())
-    if isinstance(xpath_results, (_ElementStringResult, _ElementUnicodeResult, str, Number, bool)):
+    if isinstance(xpath_results, (str, Number, bool)):
         return str(xpath_results)
     if xpath_results is None and allow_none:
         return None

